src/pages/admin/users/index.tsx

```diff
@@ -4,7 +4,7 @@ import { endpoints } from 'service/apiEndpoints'
 
 export const getServerSideProps = securedAdminProps(
   ['common', 'auth', 'person', 'admin', 'validation'],
-  () => endpoints.person.list.url,
+  () => endpoints.person.list().url,
```

Comment:
without default values it seems this will load all users - could you check and confirm if the list function needs default paging?